src/atomate2/vasp/schemas/phonons.py

```diff
@@ -0,0 +1,469 @@
+            for i, temp in enumerate(temperature_range_thermal_displacements):
+                phonon.thermal_displacement_matrices.write_cif(
+                    phonon.primitive,
+                    i,
+                    filename="tdispmat_" + str(temp) + "K.cif",
+                )
+            tdisp_mat = (
+                phonon._thermal_displacement_matrices.thermal_displacement_matrices.tolist()
+            )
+
+            tdisp_mat_cif = (
+                phonon._thermal_displacement_matrices.thermal_displacement_matrices_cif.tolist()
+            )
+
+        else:
+            tdisp_mat = None
+            tdisp_mat_cif = None
+
+        return cls(
+            structure=structure,
+            phonon_bandstructure=bs_symm_line,
+            phonon_dos=dos,
+            free_energies=free_energies,
+            temperatures=temperature_range.tolist(),
+            total_energy=total_energy / formula_units,
+            has_imaginary_modes=imaginary_modes,
+            force_constants=phonon.force_constants.tolist()
+            if kwargs["store_force_constants"]
+            else None,
+            born=borns.tolist() if borns is not None else None,
+            epsilon_static=epsilon.tolist() if epsilon is not None else None,
+            dft_phononwebsite=phononwebsite_dict,
+            supercell_matrix=phonon.supercell_matrix.tolist(),
+            primitive_matrix=phonon.primitive_matrix.tolist(),
+            code="vasp",
+            thermal_displacement_data={
+                "temperatures_thermal_displacements": temperature_range_thermal_displacements.tolist(),
+                "thermal_displacement_matrix_cif": tdisp_mat_cif,
+                "thermal_displacement_matrix": tdisp_mat,
+                "freq_min_thermal_displacements": kwargs[
+                    "freq_min_thermal_displacements"
+                ],
+            },
+            jobdirs={
+                "displacements_job_dirs": displacement_data["dirs"],
+                "static_run_job_dir": kwargs["static_run_job_dir"],
+                "born_run_job_dir": kwargs["born_run_job_dir"],
+                "optimization_run_job_dir": kwargs["optimization_run_job_dir"],
+            },
+            uuids={
+                "displacements_uuids": displacement_data["uuids"],
+                "born_run_uuid": kwargs["born_run_uuid"],
+                "optimization_run_uuid": kwargs["optimization_run_uuid"],
+                "static_run_uuid": kwargs["static_run_uuid"],
+            },
+            phonopy_settings={
+                "npoints_band": kwargs["npoints_band"],
+                "kpath_scheme": kpath_scheme,
+                "kpoint_density_dos": kwargs["kpoint_density_dos"],
+            },
+        )
+
+    @staticmethod
+    def get_kpath(
+        structure: Structure, kpath_scheme: str, symprec: float, **kpath_kwargs
+    ):
+        """
+            get high-symmetry points in k-space
+            Parameters
+        -   ---------
```

Comment:
Weird formatting?

Also there should be a blank line between the first line and the Parameters line

Reply:
Uups.